test/numa/tests/memtier/n4c16/test01-only-add-pods/code.var.sh

```diff
@@ -0,0 +1,37 @@
+CPU=1 create guaranteed
+verify \
+    'len(allowed_cpus["pod0"]) == 1'
+report allowed
+
+CPU=2 create guaranteed
+verify \
+    'len(allowed_cpus["pod0"]) == 1' \
+    'len(allowed_cpus["pod1"]) == 2' \
+    'len(allowed_cores["pod1"]) == 1' \
+    'len(allowed_cpus["pod0"].union(allowed_cpus["pod1"])) == 3'
+report allowed
+
+CPU=3 create guaranteed
+verify \
+    'len(allowed_cpus["pod0"]) == 1' \
+    'len(allowed_cpus["pod1"]) == 2' \
+    'len(allowed_cores["pod1"]) == 1' \
+    'len(allowed_cpus["pod2"]) == 3' \
+    'len(allowed_cores["pod2"]) == 2' \
+    'len(allowed_nodes["pod2"]) == 1' \
+    'len(allowed_cpus["pod0"].union(allowed_cpus["pod1"]).union(allowed_cpus["pod2"])) == 6'
```

Comment:
Maybe it would be a good idea to run this input files through a simple pre-processing script
which performs a set of simple well defined (and documented) translations on the input and
then source the resulting output. That would make it possible to implement conventions that
would allow leaving a lot of that almost-implied repetitive stuff out which I suspect will quickly
become annoying when writing tests. 

For instance, this could then become something like this, with implied leading `verify '` and
a trailing `'` for every line starting with TAB after a single `verify`. There is probably a hugton
of other similar useful conventions that can really make writing these more convenient.
```
CPU=3 create guaranteed
verify
	len(allowed_cpus["pod0"]) == 1
	len(allowed_cpus["pod0"]) == 1
	len(allowed_cpus["pod0"]) == 1
...
```